omero/sysadmins/cli/usergroup.rst

```diff
@@ -57,6 +57,33 @@ below:
 
 .. literalinclude:: /downloads/ldap/setdn.out
 
+
+User deactivation
+^^^^^^^^^^^^^^^^^
+
+To deactivate a user, remove him/her from the system group ``user``.
+Use the command :program:`omero user leavegroup` and specify the ``user`` group as the target::
+
+    # Remove jsmith from group user
+    $ bin/omero user leavegroup user --name=jsmith
+
+To reactivate the user, add him/her back to the system group ``user`` i.e.::
+
+    $ bin/omero user joingroup user --name=jsmith
+
+
+User editing
+^^^^^^^^^^^^
+
+Updating the details of a user e.g. the email address can be achieved using
+the :program:`omero obj update` command::
+
+    # Determine the ID of jsmith
+    $ bin/omero user info --user-name jsmith
```

Comment:
here, again the ``--user-name`` flag is not necessary.
See example below

```
bin/omero user info jsmith
Using session1 (user-6@eel.openmicroscopy.org:4064). Idle timeout: 10 min. Current group: private-1
 id   | login  | first name | last name | email | active | ldap  | admin | member of | owner of 
------+--------+------------+-----------+-------+--------+-------+-------+-----------+----------
 3102 | jsmith | john       | smith     |       |        | False |       | 1757      |          
(1 row)
```

Reply:
It was to make it more explicit